### PR TITLE
Actionbar vanilla messages

### PR DESCRIPTION
**Pull request by BluePsychoRanger**
This PR adds vanilla actionbar message detection, fixes some message bugs, and properly documents the library.
Requires #8 to properly build

### CODE REVIEW

File: libs/actionbar/data/smithed.actionbar/advancements/impl/__version__/vanilla/bed/clicked_bed.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
+{
+  "criteria": {
+    "unoccupied": {
+      "trigger": "minecraft:item_used_on_block",
+      "conditions": {
+        "player": [
+          {
+            "condition": "minecraft:inverted",
+            "term": {
+              "condition": "minecraft:value_check",
```
Comment (vdvman1) [inline libs/actionbar/data/smithed.actionbar/advancements/impl/__version__/vanilla/bed/clicked_bed.json:10]: Why use a score check here? Can't you use an `entity_properties` condition that checks the `is_sneaking` value in the `flags`?

Comment (vdvman1) [reply libs/actionbar/data/smithed.actionbar/advancements/impl/__version__/vanilla/bed/clicked_bed.json:10]:
```json
{
  "condition": "minecraft:entity_properties",
  "entity": "this",
  "predicate": {
    "flags": {
      "is_sneaking": true
    }
  }
}
```

Comment (BluePsychoRanger) [reply libs/actionbar/data/smithed.actionbar/advancements/impl/__version__/vanilla/bed/clicked_bed.json:10]: The scoreboard objective and the flag actually check different things. `is_sneaking` is true if the player's model is sneaking, meaning it's true when the player is force sneaking under a half slab. However the scoreboard objective is only true when the player is physically sneaking (i.e. they're holding down shift), meaning it's false when the player is force sneaking. The scoreboard objective is required to detect when the player is able to actually interact with the bed.

Comment (vdvman1) [reply libs/actionbar/data/smithed.actionbar/advancements/impl/__version__/vanilla/bed/clicked_bed.json:10]: Oh, I had no idea it was different. That's annoying
Good to know!

**Pull request merged**